### ISSUE TITLE
"phone storage is full,delete some resources"

### ISSUE THREAD

**Issue by mSv1306** [Mygod/VPNHotspot#220]
The application stops working because the storage fills up after a few hours, what can I do to fix it?

**Comment by Mygod** [comment]
Do you look at the "Clients" screen constantly? That would increase the rate of this app recording traffic stats. Try turning off the screen and see if that helps.

**Comment by Mygod** [comment]
I guess a `android:manageSpaceActivity` could be added.

**Comment by mSv1306** [comment]
Yes please, I still have to clean the data every few hours because the storage fills up completely.

**Comment by Mygod** [comment]
> Do you look at the "Clients" screen constantly? That would increase the rate of this app recording traffic stats. Try turning off the screen and see if that helps.

Did this fix help you?

**Comment by mSv1306** [comment]
No, it didn't work.

**Comment by Mygod** [comment]
How many devices are you connecting?

**Comment by mSv1306** [comment]
I'm connecting a wifi repeater and 5 devices are connected to that repeater. Is it possible that this is the cause?

**Comment by Mygod** [comment]
Ok I'll try to do something about it in the next release.

**Comment by Mygod** [comment]
I have identified a bug which could be causing increased disk usage. Please check v2.12.7 to see if this issue is resolved.

**Comment by Mygod** [comment]
[v2.12.7](https://github.com/Mygod/VPNHotspot/releases/tag/v2.12.7) has now been released.
